.github/workflows/ci_admin.yml

```diff
@@ -80,7 +80,7 @@ jobs:
       - run: bundle exec brakeman --rails6 --force-scan .
         name: Scanning Security issues on module
         working-directory: ${{ env.DECIDIM_MODULE }}
-      - run: bundle exec rspec
+      - run: bundle exec rspec --exclude-pattern 'spec/system/**/*_spec.rb'
```

Comment:
Let's change the name of this workflow to `"[CI] Admin (unit tests)"` so it's consistent